### ISSUE TITLE
HNC: document instructions with Prometheus

### ISSUE THREAD

**Issue by adrianludwin** [kubernetes-sigs/multi-tenancy#433]
This is for @yeya24 due to his work on #416 - thanks Ben!

We're about to publish a doc for how to monitor HNC with Stackdriver. If you could write a doc about how to monitor with Prometheus, that would be great! You can write it any way you like (eg Google Doc, a new section in the README, etc). It doesn't need to be long - if you only need a paragraph or two (eg "Deploy Prometheus using operator X, and modify this resource to enable scraping") then that's even better.

**Comment by yeya24** [comment]
Thanks for the issue. Feel free to assign this to me. But I am busy this week. So maybe I can work on the doc this weekend. Is it ok?

**Comment by adrianludwin** [comment]
Yup! You're not a member of kubernetes-sigs so I can't actually assign the
issue to you but you can treat it as though it's yours. Thanks!

**Comment by adrianludwin** [comment]
See #435 as a good model for this kind of doc.

**Comment by adrianludwin** [comment]
Hi @yeya24 , are you still planning to work on this issue? Thanks!

**Comment by adrianludwin** [comment]
@yeya24 Hi Ben, another update, are you planning on writing this? Thanks!
